src/Symfony/Bundle/FrameworkBundle/Resources/views/Form/widget_attributes.html.php

```diff
@@ -1,3 +1,3 @@
-id="<?php echo $view->escape($id) ?>" name="<?php echo $view->escape($full_name) ?>"<?php if ($disabled): ?> disabled="disabled"<?php endif ?>
-<?php if ($required): ?> required="required"<?php endif ?>
-<?php echo $attr ? ' '.$view['form']->block($form, 'attributes') : '' ?>
+id="<?php echo $view->escape($id); ?>" name="<?php echo $view->escape($full_name); ?>"<?php if ($disabled): ?> disabled="disabled"<?php endif; ?>
```

Comment:
these changes should be reverted

Reply:
These changes are required by the fabpot.io bot... If we don't do it, it screams...
In the previous version, we've edit something in this file, and fix it to avoid it. Now, it's true, we have remove modification from this file...

Reply:
Oh, okay, I patch it.